modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/product/navigation/control/menu/LayoutReportsProductNavigationControlMenuEntry.java

```diff
@@ -271,9 +269,11 @@ private boolean _isEmbeddedPersonalApplicationLayout(Layout layout) {
 	}
 
 	private boolean _isPanelStateOpen(HttpServletRequest httpServletRequest) {
-		String keepAuditPanelOpen  = httpServletRequest.getParameter("keepAuditPanelOpen");
+		String keepAuditPanelOpen = httpServletRequest.getParameter(
+			"keepAuditPanelOpen");
+
+		if (StringUtil.equalsIgnoreCase(keepAuditPanelOpen, "true"))
```

Comment:
This would be much simpler by using:

```
if (ParamUtil.getBoolean(httpServletRequest, "keepAuditPanelOpen"))
```

We try to always use `ParamUtil`.